frontend/views/containers/notifications/NotificationsList.vue

```diff
@@ -0,0 +1,250 @@
+  name: 'NotificationsList',
+  props: {
+    variant: {
+      type: String,
+      validator: (value) => ['compact', 'default'].includes(value),
+      default: 'defaut'
+    }
+  },
+  data: () => ({
+    config: {
+      notifications: null
+    },
+    ephemeral: {
+      isLoading: false
+    }
+  }),
+  components: {
+    Avatar
+  },
+  mounted () {
+    // TODO this (All dumb logic for now)
+
+    // Simulate view with no notifications
+    if (this.notificationsCount === 0) {
+      return null
+    }
+
+    // Simulate view with only 1 notification
+    if (this.notificationsCount === 1) {
+      this.ephemeral.isLoading = true
+      setTimeout(() => {
+        // handpick a notification that is "MEMBER_ADDED"
+        this.config.notifications = {
+          3213: fakeDBWithNotifications[3213]
+        }
+        this.ephemeral.isLoading = false
+      }, 500)
+      return
+    }
+
+    // Load notifications from fake DB
+    this.ephemeral.isLoading = true
+    setTimeout(() => {
+      this.config.notifications = fakeDBWithNotifications
+      this.ephemeral.isLoading = false
+    }, 1500)
+  },
+  computed: {
+    ...mapGetters([
+      'notificationsCount'
+    ]),
+    dateNow () {
+      // Hardcoded so the dummy layout makes sense
+      return 1590823007327
+    },
+    notifications () {
+      if (!this.config.notifications) {
+        return {}
+      }
+
+      const list = {}
+      const listNew = []
+      const listOlder = []
+
+      for (const nId in this.config.notifications) {
+        const { timestamp, type, data } = this.config.notifications[nId]
+        const listToPush = this.dateNow - timestamp < HOURS_MILLIS * 2 ? listNew : listOlder
+        listToPush.push(templates[type]({ data, timestamp }))
+      }
+
+      if (listNew.length > 0) {
+        list.new = listNew
+      }
+
+      if (listOlder.length > 0) {
+        list.older = listOlder
+      }
+
+      return list
+    }
+  },
+  methods: {
+    title: (type) => {
+      return {
+        new: L('New'),
+        older: L('Older')
+      }[type]
+    },
+    iconBg: (level) => {
+      return {
+        info: 'has-background-primary has-text-primary',
+        success: 'has-background-success has-text-success',
+        danger: 'has-background-danger has-text-danger'
+      }[level]
+    }
+  }
+}
+</script>
+
+<style lang="scss" scoped>
+@import "@assets/style/_variables.scss";
+
+.c-empty {
+  padding: 3rem 1rem;
+  text-align: center;
+}
+
+.c-loading {
+  .compact & {
+    padding: 0 1rem;
+  }
+}
+
+.c-skeleton {
+  display: flex;
+  align-items: center;
+  padding: 1rem 0;
+
+  &-circle {
+    display: block;
+    width: 2.5rem;
+    height: 2.5rem;
+    border-radius: 50%;
+    margin-right: 0.5rem;
+    background-color: $general_2;
+  }
+
+  &-line {
+    flex-grow: 1;
+    height: 1.5rem;
+    border-radius: 0.7rem;
+    background-color: $general_2;
+  }
+}
+
+.c-title {
+  display: block;
+  padding: 0 0 0.5rem 0.5rem;
```

Comment:
I think changing the left padding here to 0 is enough to fix the alignment issue.

Reply:
yes it did. updated